src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -435,6 +459,7 @@ namespace wallet_rpc
       uint64_t below_amount;
       bool do_not_relay;
       bool get_tx_hex;
+      bool get_tx_metadata;
```

Comment:
The same change seems necessary for the recently added `sweep_single` command as well.

Reply:
Unrelated to this patch, it's another that I cherry-picked because it's waiting to be merged. I'll make the change there.

Reply:
https://github.com/monero-project/monero/pull/2788